### PR TITLE
Changes from ticket patch.

### PR DESCRIPTION
**Pull request by adamsilverstein**
Trac ticket: 

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/functions.php [hunk 1]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ * Sanitizes an attributes array into an attributes string to be placed inside a `<script>` tag.
+ *
+ * Automatically injects type attribute if needed.
+ * Used by {@see wp_get_script_tag()} and {@see wp_get_inline_script_tag()}.
+ *
+ * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7791]:
```suggestion
 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 2]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+	return $attributes_string;
+}
+
+/**
+ * Formats `<script>` loader tags.
+ *
+ * It is possible to inject attributes in the `<script>` tag via the {@see 'wp_script_attributes'} filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7821]:
```suggestion
 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 3]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+ * @return string String containing `<script>` opening and closing tags.
+ */
+function wp_get_script_tag( $attributes ) {
+	if ( ! isset( $attributes['type'] ) && ! is_admin() && ! current_theme_supports( 'html5', 'script' ) ) {
+		$attributes['type'] = 'text/javascript';
+	}
+	/**
+	 * Filters attributes to be added to a script tag.
+	 *
+	 * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7833]:
```suggestion
	 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 4]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+	return sprintf( "<script%s></script>\n", wp_sanitize_script_attributes( $attributes ) );
+}
+
+/**
+ * Prints formatted `<script>` loader tag.
+ *
+ * It is possible to inject attributes in the `<script>` tag via the  {@see 'wp_script_attributes'}  filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7850]:
```suggestion
 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 5]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+	echo wp_get_script_tag( $attributes );
+}
+
+/**
+ * Wraps inline JavaScript in `<script>` tag.
+ *
+ * It is possible to inject attributes in the `<script>` tag via the  {@see 'wp_script_attributes'}  filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7864]:
```suggestion
 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 6]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+ * @return string String containing inline JavaScript code wrapped around `<script>` tag.
+ */
+function wp_get_inline_script_tag( $javascript, $attributes = array() ) {
+	if ( ! isset( $attributes['type'] ) && ! is_admin() && ! current_theme_supports( 'html5', 'script' ) ) {
+		$attributes['type'] = 'text/javascript';
+	}
+	/**
+	 * Filters attributes to be added to a script tag.
+	 *
+	 * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7877]:
```suggestion
	 * @since 5.7.0
```

File: src/wp-includes/functions.php [hunk 7]
```diff
@@ -7781,3 +7781,123 @@ function is_php_version_compatible( $required ) {
...
+	return sprintf( "<script%s>%s</script>\n", wp_sanitize_script_attributes( $attributes ), $javascript );
+}
+
+/**
+ * Prints inline JavaScript wrapped in `<script>` tag.
+ *
+ * It is possible to inject attributes in the `<script>` tag via the  {@see 'wp_script_attributes'}  filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @since 5.6.0
```
Comment (hellofromtonya) [inline src/wp-includes/functions.php:7896]:
```suggestion
 * @since 5.7.0
```

**Comment by adamsilverstein** [comment]
Thanks for the review @hellofromtonya - I fixed these in https://github.com/WordPress/wordpress-develop/pull/965 - sorry for the confusion :)

**Pull request closed without merging**